### PR TITLE
system mount points for FCoE and iSCSI in deployment guide

### PR DESCRIPTION
**Pull request by sven15**
bsc#1010297 and bsc#1012335#c28

### CODE REVIEW

**Comment by sven15** [comment]
[add NOTE: that separate /var and /usr partitions not supported for FCoE network boot](https://trello.com/c/47Tv0uBD/389-add-note-that-separate-var-and-usr-partitions-not-supported-for-fcoe-network-boot)

File: xml/inst_yast2.xml [hunk 1]
```diff
@@ -2392,7 +2392,17 @@ sle_update.xml#sec.update.terminology: "Modules" -->
     </listitem>
    </itemizedlist>
   </note>
-
+  <note>
+   <title>Mount Points for FCoE and iSCSI Devices</title>
+   <para>FCoE and iSCSI devices will appear asynchronously during the
+    boot process. While the initrd guarantees that those devices are
+    setup correctly for the root file system, there are no such
```
Comment (sknorr) [inline xml/inst_yast2.xml]: style: setup -> set up (this is a verb)

File: xml/inst_yast2.xml [hunk 2]
```diff
@@ -2392,7 +2392,17 @@ sle_update.xml#sec.update.terminology: "Modules" -->
...
+    guarantees for any other file systems or mount points like 
+    <filename>/usr</filename>. Hence any system mount points like 
+    <filename>/usr</filename> or <filename>/var</filename> are not 
+    supported. If you want to use those devices, ensure correct
+    synchronisation with the respective services and devices.</para>
```
Comment (sknorr) [inline xml/inst_yast2.xml]: Sounds a bit circular: "to use devices ... ensure sync with devices." - not sure here.

Comment (sven15) [inline xml/inst_yast2.xml]: Yes, an "of" would be better.

**Pull request merged**